code/modules/reagents/reagent_dispenser.dm

```diff
@@ -41,7 +41,7 @@
 //Dispensers
 /obj/structure/reagent_dispensers/watertank
 	name = "watertank"

```

Comment:
To quote example code in this very PR:
`name = "power cable"`
A space is perfectly fine and the only thing that would break would be anything that doesn't use path name over name name (i.e. bad code that should be fixed regardless).